lib/create-environment-api.js

```diff
@@ -32,6 +32,8 @@ import entities from './entities'
  * @prop {function} getUiExtensions
  * @prop {function} createUiExtension
  * @prop {function} createUiExtensionWithId
+ * @prop {function} getAppInstallation
+ * @prop {function} getAppInstallations
```

Comment:
we're missing `createAppInstallation` here :)

Reply:
@FLoppix

Reply:
nice catch @phoebeschmidt  :D